### ISSUE TITLE
The API used by StockholmProvider.java has been discontinued...

### ISSUE THREAD

**Issue by regagain** [schildbach/public-transport-enabler#83]
...since April 1st, as explained [here](http://www.trafiken.nu/stockholm/arkiv/2016/reseplaneraren-avvecklad/) (in Swedish). As a result, all queries for this provider fail.

They recommend using http://sl.se/ or http://resrobot.se/ instead.

**Comment by grote** [comment]
The question is if the old API and the new API are the same, so that just the Query URL needs to be changed. On their side, I made a request and this URL was called:
```
http://sl.se/api/sv/TravelPlanner/SearchTravelById/Karl%20XII;s%20torg%20(Stockholm)/Torsvik%20(Liding%C3%B6)/1023/9252/null/depart/sv/null/null/2,8,1,4,96,/null/null/null/null/null/false/null/0/0/null/false
```

**Comment by schildbach** [comment]
StockholmProvider is Hafas-based. So it won't be able to read the API @grote suggests.

I wonder if SeProvider already has all the data we needed. It uses `http://samtrafiken.hafas.de/bin/`, and according to my comments at some point in time (months ago) both `http://reseplanerare.resrobot.se/bin/` and `http://api.vasttrafik.se/bin/` seemed to point to the same API.

**Comment by regagain** [comment]
I checked today & yes, SeProvider has all the Stockholm data.

PS: I apologize for the late reply.

**Comment by schildbach** [comment]
Fixed via 26621e5.
